cross_compile/ros2_cross_compile.py

```diff
@@ -103,6 +103,14 @@ def create_arg_parser():
              'adding extra apt sources that rosdep may not handle, or other arbitrary setup that '
              'is specific to your application build. See the section on "Custom Setup Script" '
              'in the README.md for more details.')
+    parser.add_argument(
+        '--custom-data-dir',
+        required=False,
+        default=None,
+        type=str,
+        help='Provide a path to a custom arbitrary to make available the sysroot container. '
+             'You may use this data in your --custom-setup-script. '
+             'It will be available as "custom_data/" in the working directory.')
```

Comment:
working directory of the docker container?

Reply:
Current working directory when the custom-setup-script is run. What do you think of this wording?

```
You may use this data in your --custom-setup-script, it will be available as "./custom_data/" in the current working directory when the script is run.
```